lib/jnpr/junos/device.py

```diff
@@ -1025,7 +1025,7 @@ def transform(self, func):
     # -----------------------------------------------------------------------
 
     def __new__(cls, *args, **kwargs):
-        if kwargs.get('port') in [23, '23'] or kwargs.get('mode'):
+        if kwargs.get('port') in [22, '22', 23, '23'] or kwargs.get('mode'):
```

Comment:
This condition is not correct. Even we can use port 22 to connect directly to the device (when default 830 is not configured).